tests/clpy_tests/opencl_tests/ultima_tests/test_template.py

```diff
@@ -0,0 +1,163 @@
+# flake8: noqa
+# TODO(vorj): When we will meet flake8 3.7.0+,
+#               we should ignore only W291 for whole file
+#               using --per-file-ignores .
+
+import unittest
+from utility import _exec_ultima
```

Comment:
Why don't you use `utility._exec_ultima` directly? `unittest` uses `unittest.TestCase` thus `utility._exec_ultima` is better.

Reply:
It was fixed at cd1fc0e.